examples/text_labels_and_annotations/text_alignment.py

```diff
@@ -1,13 +1,44 @@
 """
-===================
-Precise text layout
-===================
+==============
+Text alignment
+==============
 
-You can precisely layout text in data or axes coordinates.  This example shows
-you some of the alignment and rotation specifications for text layout.
+Texts are aligned relative to their anchor point depending on the properties
+``horizontalalignment`` and ``verticalalignment``.
 """
 
 import matplotlib.pyplot as plt
+import numpy as np
+
+y = [0.22, 0.34, 0.5, 0.56, 0.77]
+x = [0.17, 0.5, 0.855]
```

Comment:
I've determined the anchor point positions empirically and it seems to be good enough. Does anybody know how to transform `Renderer.get_text_width_height_descent()` from display coordinates to axes coordinates?

Reply:
Something like `ax.transAxes.inverted().transfrom*`?

Reply:
I don't get it:
```
import matplotlib.pyplot as plt
import numpy as np

fig, ax = plt.subplots(dpi=100)
ax.set(xlim=(0, 1), ylim=(0, 1), xticks=[], yticks=[])
t = ax.text(0.5, 0.5, 'plot', fontsize=128, ha='center', va='center', zorder=1)
ax.spines[:].set_visible(False)

plt.draw_if_interactive()
width, height, descent = fig.canvas.get_renderer().get_text_width_height_descent('plot', t.get_font_properties(), ismath=False)

(w, h), = ax.transAxes.inverted().transform([(width, height)])
x = [0.5 - w/2, 0.5, 0.5 + w/2]
y = [0.5 - h/2, 0.5, 0.5 + h/2]

ax.vlines(x, y[0], y[-1], color='grey')
ax.hlines(y, x[0], x[-1], color='grey')
```

![grafik](https://user-images.githubusercontent.com/2836374/141536931-eb9898ea-40ee-4a81-8d4c-97565e7eec69.png)

I thought that shoud bound the text.

Reply:
`vlines` and `hlines` are in data space? Then you'd need `transData`.

Reply:
🐑 Thanks!

Reply:
Still does not work. Btw. `transData` and `transAxes` are the same since `ax.set(xlim=(0, 1), ylim=(0, 1))`.